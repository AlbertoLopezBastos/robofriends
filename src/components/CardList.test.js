import { shallow } from 'enzyme';
import CardList from './CardList';

it('Expect to render CardList Component', () => {
  const mockRobots = [{ id: 1, name: 'John Jode', username: 'JohnJohn', enail: 'john@gmail.com' }]
  expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot();
})
