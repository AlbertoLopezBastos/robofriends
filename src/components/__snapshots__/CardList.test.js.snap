// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render CardList Component 1`] = `
<div>
  <Card
    id={1}
    key="1"
    name="John Jode"
  />
</div>
`;
